pywr-core/src/parameters/interpolate.rs

```diff
@@ -104,7 +104,7 @@ mod tests {
         let not_enough_points = vec![(1.0, 3.0)];
         assert!(linear_interpolation(1.0, &not_enough_points, true).is_err());
 
-        let non_monotonic_points = vec![(1.0, 3.0), (2.0, 4.5), (2.0, 6.0), (4.0, 7.5), (5.0, 9.0)];
+        let _non_monotonic_points = vec![(1.0, 3.0), (2.0, 4.5), (2.0, 6.0), (4.0, 7.5), (5.0, 9.0)];
```

Comment:
Shouldn't this be used in the asset below?

Reply:
Yes! Fixed now.